app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -189,34 +189,65 @@
           </h3>
         {{/if}}
       </div>
-      <button type="button" class="ui blue small button" {{action 'addTicket' 'free' data.event.tickets.length}}>
-        <i class="large icons basic-details">
-          <i class="smile icon"></i>
-          <i class="inverted corner add icon"></i>
-        </i>
-        {{t 'Free Ticket'}}
-      </button>
-      <button type="button" class="ui blue small button" {{action 'addTicket' 'paid' data.event.tickets.length}}>
-        <i class="large icons basic-details">
-          <i class="dollar icon"></i>
-          <i class="inverted corner add icon"></i>
-        </i>
-        {{t 'Paid Ticket'}}
-      </button>
-      {{!-- {{else}}
-        <div class="field">
-          <label for="ticket_url">{{t 'Ticket URL'}}</label>
-          {{widgets/forms/link-input inputId='ticket_url' segmentedLink=data.event.segmentedTicketUrl}}
-        </div>
-      {{/if}} --}}
+      {{#unless device.isMobile}}
```

Comment:
This is not optimal solution , this costs too much in FE optimisation and loading time. Also, try to solve using some small variations . Such small change should not incur changes of more than 50 loc